src/parsing/parser.cc

```diff
@@ -3164,12 +3385,9 @@ void Parser::HandleSourceURLComments(LocalIsolate* isolate,
   }
 }
 
-template void Parser::HandleSourceURLComments(Isolate* isolate,
-                                              Handle<Script> script);
-template void Parser::HandleSourceURLComments(OffThreadIsolate* isolate,
-                                              Handle<Script> script);
```

Comment:
wasn't sure how to resolve these two. deleted to get the code compiling in the meantime. any suggestions?

Reply:
What was the issue?